pco_python/src/lib.rs

```diff
@@ -50,6 +50,79 @@ pub fn pco_err_to_py(pco: PcoError) -> PyErr {
   PyRuntimeError::new_err(format!("pco error: {}", pco))
 }
 
+#[pyclass(name = "IntMultSpec")]
+#[derive(Clone, Default)]
+pub struct PyIntMultSpec(IntMultSpec);
+
+/// Specifies how pcodec should handle modulo compression for integer types.
+#[pymethods]
+impl PyIntMultSpec {
+  /// :returns: a IntMultSpec disabling modulo compression.
+  #[staticmethod]
+  fn disabled() -> Self {
+    Self(IntMultSpec::Disabled)
+  }
+
+  /// :returns: a IntMultSpec enabling modulo compression.
```

Comment:
Let's say that this will automatically try to detect whether to use the mode and which `base` to use? (we should start doing this in the Rust docs too)

Reply:
Changing it to
```
  /// :returns: a IntMultSpec enabling modulo compression. This will
  /// automatically try to detect whether to use the mode and which `base` to
  /// use.